src/fragments/Markdown.react.js

```diff
@@ -2,12 +2,19 @@ import React, {Component} from 'react';
 import {type} from 'ramda';
 import Markdown from 'react-markdown';
 
+import MarkdownHighlighter from '../utils/MarkdownHighlighter';
 import {propTypes, defaultProps} from '../components/Markdown.react';
 import '../components/css/highlight.css';
```

Comment:
If we're to keep this custom css, might as well import it when we request hljs instead of getting it now.